### PR TITLE
Chooser test apis

### PR DESCRIPTION
**Pull request by jyasskin**
These were implemented in Chrome's testing library in https://codereview.chromium.org/1325953002/. Demo at https://rawgit.com/jyasskin/web-bluetooth-1/chooser-test-apis/tests/index.html.

### CODE REVIEW

File: tests/index.bs [hunk 1]
```diff
@@ -208,6 +214,71 @@ read or written, the characteristic with UUID <code>errorUUID(which)</code>
...
+
+: The UA discovers that a device with {{BluetoothDevice/id}}
+    <var>device-id</var> is no longer nearby.
+:: <code>"remove-device(<var>device-id</var>)"</code>
+
+Note: {{TestRunner/getBluetoothManualChooserEvents()}} returns a
+<code>sequence&lt;DOMString></code> rather than the
+<code>{{Promise}}&lt;sequence&lt;DOMString>></code> that we'd normally prefer,
+for two reasons. First, the Chromium implementation has trouble returning a
+{{Promise}} from this interface, for bad reasons. Second, we aren't providing a
```
Comment (scheib) [inline tests/index.bs]: ;) The second reason is a bit weak (we could of course have send...Event return a promise if the first reason wasn't an issue). I think the first reason is valid enough.

**Comment by scheib** [comment]
LGTM, though the one comment could be shortened as noted.

File: tests/index.bs (hunk 1 above)
Comment (jyasskin) [inline tests/index.bs]: Fair enough. How's this?

Comment (scheib) [inline tests/index.bs]: LGTM

**Pull request merged**